manifest/v1alpha/report/validation.go

```diff
@@ -0,0 +1,110 @@
+package report
+
+import (
+	"github.com/pkg/errors"
+
+	validationV1Alpha "github.com/nobl9/nobl9-go/internal/manifest/v1alpha"
+	"github.com/nobl9/nobl9-go/internal/validation"
+	"github.com/nobl9/nobl9-go/manifest"
+	"github.com/nobl9/nobl9-go/manifest/v1alpha"
+)
+
+const (
+	expectedNumberOfReportTypes = 1
+)
```

Comment:
I'm absolutely fine with not having this `const`, and simply checking `if reportTypeCounter != 1`, it's not like this will eventually be a configuration option (I think). Error message is really clear what's expected there.